generator.py

```diff
@@ -0,0 +1,16 @@
+# Main bottle route generator 
+
+from bottle import route, run, static_file, Bottle
+
+main = Bottle()
+
+@main.route('/')
+def index():
+	return static_file('main.html', root='/home/brad/Documents/Bottle/WebApi/RoboticsWebApi')
```

Comment:
It's considered bad practice to  hard code local paths in your code, like `/home/brad/`. Does bottle have a config module?